src/meta/src/stream/scale.rs

```diff
@@ -1933,11 +1999,11 @@ impl ScaleController {
                                         dispatcher.dispatcher_id as FragmentId
                                     );
                                 } else {
-                                    bail!(
-                                        "downstream actor id {} from actor {} not found in fragment_actor_id_map",
-                                        downstream_actor_id,
-                                        actor.actor_id,
-                                    );
+                                    // bail!(
+                                    //     "downstream actor id {} from actor {} not found in actor_fragment_id_map_for_check: {actor_fragment_id_map_for_check:?}",
+                                    //     downstream_actor_id,
+                                    //     actor.actor_id,
+                                    // );
```

Comment:
This fails when `ALTER PARALLELISM` for downstream SourceBackfill's MV. Still investigating. 🤔

Reply:
I found the problem. This happens when I commented out the last `DROP SOURCE` in the test, and rely on `risedev slt-clean` (which `DROP DATABASE`). But there's a bug in `DROP DATABASE` that the source streaming job not dropped, thus this line fails (when ALTER any job).

Reply:
fix in #18033